src/native/crt.c

```diff
@@ -47,6 +49,151 @@ struct aws_allocator *aws_jni_get_allocator() {
     return s_allocator;
 }
 
+void s_detach_jvm_from_thread(void *user_data) {
+    AWS_LOGF_DEBUG(AWS_LS_COMMON_GENERAL, "s_detach_jvm_from_thread invoked");
+    JavaVM *jvm = user_data;
+    (*jvm)->DetachCurrentThread(jvm);
```

Comment:
Seems like a crash can still happen here. Shouldn't we check here if the jvm has already exited before we try and use it?